src/app/pages/charts/charts.module.ts

```diff
@@ -7,6 +7,8 @@ import { routing }       from './charts.routing';
 import { Charts } from './charts.component';
 import { ChartistJs } from './components/chartistJs/chartistJs.component';
 import { ChartistJsService } from './components/chartistJs/chartistJs.service';
+import {MorrisComponent} from "./components/morrisJs/morris.componenet";
```

Comment:
Please make sure you used correct style guide (spaces around the component name).